.claude/skills/fetch-and-automate-workflow.md

```
@@ -0,0 +1,268 @@
+
+Match `<id>` argument against patterns:
+
+- **Polarion**: `^(OSE|OCP)-\d+$`
+- **Jira**: `^(MCO|OCPBUGS)-\d+$`
+
+If no match, show error and exit.
+
+### 2. Fetch from Polarion (if Polarion ID)
+
+**Check MCP availability:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)